controllers/UserManagement/ManageOrgsController.js

```diff
@@ -0,0 +1,35 @@
+const BaseController = require('../BaseController');
+const { orgLookup } = require('../../services/UserManagement/OrganisationSearchService');
+
+class ManageOrgsController extends BaseController {
+  async getValues(req, _res, next) {
+    try {
+      const orgsResult = await orgLookup({
+        ...this.getOptions(req),
+        url: '/admin/organisations'
+      });
+
+      const orgs = orgsResult.organisations;
+
+      if (orgs.length == 1) {
+        return _res.redirect(`/admin/organisations/${orgs[0].id}`);
+      } else {
```

Comment:
You can reduce the cyclomatic complexity here by removing the `else` as the first conditional block `return`s the function:

```
if (orgs.length == 1) {
    return _res.redirect(`/admin/organisations/${orgs[0].id}`);
}
req.sessionModel.set('orgsResult', orgs);
next();
```

Reply:
Done the changes